.gitignore

```diff
@@ -59,3 +59,4 @@ uImage
 .dirlinks
 .vscode
 .DS_Store
+nuttx Default (copy).launch

```

Comment:
I'm not sure if we should commit this

Reply:
Nope - sometimes my git commits include this and I try to filter them out when doing a PR. Missed this one, but when its wrapped up with other commits its not so easy to undo it. I'll work it out.